### ISSUE TITLE
Further questions to the previous issue

### ISSUE THREAD

**Issue by MaverickRen** [dvlab-research/LISA#42]
Thanks for helping me out with this issue(issues/41). 
Is v0 the version before the code rewrite on August 23rd? I actually use v0, so if I want to reproduce the non-ft results in the paper, I have to remove the ReasonSeg data set from the training set, right?
To reproduce the results of the ft version in the paper, how should I configure the fine-tuning on ReasonSeg, including the number of training steps and learning rate?

**Comment by X-Lai** [comment]
Yes, you're correct. You should remove the reason_seg dataset for the non-ft results. To reproduce the results in our paper, we use the datasets including ADE20K, coco-stuff, part_seg, lvis_part, and refcoco series. We do not use mapillary during training. Using these datasets during training could reproduce the results reported in the paper. 

Besides, to ensure that you can reproduce totally the same results, I will release the original model that can reproduce the results reported in the paper soon. I am sorry to mislead you previously. Please stay tuned.
